package.json

```diff
@@ -105,6 +108,8 @@
     "test:e2e:dev": "env BASE_URL=http://127.0.0.1:8000/BookReaderDemo/ npx testcafe --live --dev",
     "test:watch": "npx jest --watch",
     "test:coverage": "npx jest --coverage --colors",
-    "codecov": "npx codecov"
+    "codecov": "npx codecov",
+    "test:karma": "karma start --coverage",
+    "test:watch-karma": "karma start --auto-watch=true --single-run=false"
```

Comment:
Let's follow the pattern we have with `build`, so let's have scripts `test-jest`, `test-karma`, `test-jest:watch`, `test-karma:watch`, `test-e2e`, `test-e2e:watch`. And probably `test` and `test:watch` to run the non-e2e tests together (using npx concurrently, like build does above).